install/kubernetes/helm/istio/values.yaml

```diff
@@ -446,3 +446,39 @@ kiali:
   dashboard:
     username: admin
     password: admin
+
+certmanager:
+  enabled: false
+  replicaCount: 1
+  image:
+    repository: quay.io/jetstack/cert-manager-controller
+    tag: v0.3.1
+    pullPolicy: IfNotPresent
+  clusterResourceNamespace: "istio-system"
+  leaderElection:
+    # Override the namespace used to store the ConfigMap for leader election
+    namespace: ""
+  certificateResourceShortNames: ["cert", "certs"]
+  extraArgs: []
+  serviceAccountName: certmanager
+  resources: {}
+    # limits:
+    #  cpu: 100m
+    #  memory: 128Mi
+    # requests:
+    #   cpu: 10m
+    #   memory: 32Mi
+  podAnnotations: {}
+    # sidecar.istio.io/inject: "true"
```

Comment:
Why ?

Reply:
Well - it is already removed, since it's commented out and istio-system is excluded from auto-injection  so it had no effect even if you added it. If you want the communication between gateway and certmanager to not be encrypted - you can use a policy (mixer has one for example)